airbyte-protocol/protocol-models/src/main/resources/airbyte_protocol/well_known_types.yaml

```diff
@@ -0,0 +1,74 @@
+# Regexes are purely illustrative and need to be workshopped: BC dates shouldn't require 4-digit years; years may have >=5 digits; etc
+definitions:
+  String:
+    type: string
+    description: Arbitrary text
+  BinaryData:
+    type: string
+    description: >
+      Arbitrary binary data. Represented as base64-encoded strings in the JSON transport.
+      In the future, if we support other transports, may be encoded differently.
+    # All credit to https://stackoverflow.com/a/475217 for this pattern
+    pattern: ^(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?$
+  Date:
+    type: string
+    # Examples:
+    #   2022-01-23
+    #   2022-01-23 BC
+    # format: date is a superset of what we want, so we cannot use it here (e.g. it accepts 2-digit years)
+    pattern: ^\d{4}-\d{2}-\d{2}( BC)?$
+    description: RFC 3339§5.6's full-date format
+  TimestampWithTimezone:
+    type: string
+    # Examples:
+    #   2022-01-23T01:23:45Z
+    #   2022-01-23T01:23:45.678-11:30 BC
```

Comment:
Can TZ be UTC or EDT other wellknow TZ?

Reply:
just checked the RFC, it is only about offset. Thanks

Reply:
yeah, the other letter-codes can be ambiguous. e.g. CST being both central standard time (UTC-6) and china standard time (UTC+8)